src/components/Conversation/ViewPointCard.tsx

```diff
@@ -0,0 +1,22 @@
+import { ViewPoint } from "@/types/conversations.types";
+import ContentCard from "./ContentCard";
+import FactListCard from "./FactListCard";
+
+type ViewPointCardProps = {
+    viewpoint: ViewPoint;
+};
+
+export default function ViewPointCard({ viewpoint }: ViewPointCardProps) {
+    return (
+        <div className="flex">
+            {/* content */}
+            <div className="float-left w-9/12">
```

Comment:
The float should be unnecessary given the flexbox styling, are you sure its required?

Reply:
The right part (Fact list) is required for it to grow to the same height as the left part (Content)